processor/probabilisticsamplerprocessor/metadata.yaml

```diff
@@ -0,0 +1,8 @@
+type: probabilistic_sampler
+
+status:
+  class: processor
+  stability:
+    beta: [traces]
+    alpha: [logs]
+  distributions: [core, contrib, observiq, splunk, sumo]
```

Comment:
Should `contrib` be before `core` for alphabetical order, or do we specifically always want `core` and `contrib` as the first two entries before other distributions are listed? (I think it's best to have the primary distributions first before company-specific ones, just wanted to see if it's intentional or not).

Just looking for the best way to have consistency here.

Reply:
In other places we've always put core before contrib, i had the same question, but I'm not sure if it's important they be in alphabetical order 🤷🏻

Reply:
We generate the README according to a predefined order: core, contrib, then alphabetical.